windows-gaming-driver/src/setup/wizard.rs

```diff
@@ -249,12 +325,62 @@ impl<'a> Wizard<'a> {
             println!("");
         }
 
-        // TODO: Add a udev picker step here (just like the one we have for gpu) that selects USB keyboard and mouse.
-        // Then save that to the config and create a udev rule file that runs setfacl so the vfio group can use them.
+        // select mouse and keyboard
+        println!("Step 2: Select USB Devices");
+        println!("To enable input devices we use USB passthrough triggered by keyboard shortcuts.");
+        println!();
+        if !machine.usb_devices.is_empty() {
+            println!("You have currently selected the following usb devices: ");
+            for &(vendor, product) in machine.usb_devices.iter() {
+                println!("    {:04x}:{:04x}", vendor, product);
+            }
+            if ask::yesno(&mut self.stdin, "Do you want to remove them before proceeding?") {
+                machine.usb_devices.clear();
+                println!("Removed.");
+            }
+        }
+        println!("Please select your mouse:");
+        let mouse = self.udev_pick_usb(Some(HidKind::Mouse), &machine.usb_devices, true).expect("Failed to select Mouse");
+        println!("Please select your keyboard:");
+        let keyboard = self.udev_pick_usb(Some(HidKind::Keyboard), &machine.usb_devices, true).expect("Failed to select Keyboard");
+        if let Some(id) = mouse {
+            machine.usb_devices.insert(0, id);
+        } else {
+            println!("No mouse selected. Please select your pointing device:");
+            let mouse = self.udev_pick_usb(None, &machine.usb_devices, !machine.usb_devices.is_empty()).expect("Failed to select Pointing Device");
+            if let Some(id) = mouse {
```

Comment:
actually I think we should insist that there's a device

i guess you did it this way in case I already have something else preselected but see my prev comments on that